open-api-spec.yaml

```diff
@@ -431,6 +415,142 @@ paths:
           in: query
           name: cursor
           description: The cursor key to list from
+  /quotes:
+    post:
+      summary: Create a Quote
+      tags: []
+      operationId: create-quote
+      responses:
+        '202':
```

Comment:
Should a quote have `state` instead of returning 202 before it's ready?

Reply:
No state since quote request will be synchronous
https://github.com/interledger/open-payments/pull/97/commits/3f50d4881a4177cfe9b14366453d9fd95ba365a4